Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -1991,7 +1991,14 @@ private ChapterInfo GetChapter(IReadOnlyList<IResultSetValue> reader, BaseItem i
 
                 if (!string.IsNullOrEmpty(chapter.ImagePath))
                 {
-                    chapter.ImageTag = ImageProcessor.GetImageCacheTag(item, chapter);
+                    try
+                    {
+                        chapter.ImageTag = ImageProcessor.GetImageCacheTag(item, chapter);
+                    }
+                    catch
```

Comment:
can we catch something more specific instead of this blanket-all? or at least log the error?

Reply:
No idea what would be thrown, but I can add logging